README.md

```diff
@@ -63,24 +63,30 @@ Log in to GitHub and [register a new personal access token](https://github.com/s
 
 #### Next, Create a file named .env, copy the contents of the .env.template file into it, and add your values to the new file.
 - Paste the access token into GITHUB_TOKEN
-- TIMEZONE allows you to specify when your day begins and ends. This must be a value from the [IANA Time Zone Database](https://en.wikipedia.org/wiki/List_of_tz_database_time_zones). Typically, this would either be "Etc/UTC" or the time zone of your organization's main office, such as "America/Los_Angeles". 
+- TIMEZONE allows you to specify when your day begins and ends. This must be a value from the [IANA Time Zone Database](https://en.wikipedia.org/wiki/List_of_tz_database_time_zones). TIt would normally be "local" or the time zone of the organization's main office, such as "America / Los_Angeles", or even "utc" for the system's local zone. 
+
+
+Luxon shows some examples of possible entries, according to this table:
+
+![image](https://imgur.com/KT5jR9T.png)
+
+If for some reason you want the time to be a constant offset from UTC, you can say : "UTC+6", to mean UTC-0600. **Note that positive values in the TIMEZONE string will result in negative UTC offsets** (that is, West of UTC), while **negative values will result in positive UTC offsets.** This is a result of IANA, not by choice. 
 
-If for some reason you want the time to be a constant offset from UTC, you can say : "Etc/GMT+6", to mean UTC-0600. **Note that positive values in the TIMEZONE string will result in negative UTC offsets** (that is, West of UTC), while **negative values will result in positive UTC offsets.** This is a result of IANA, not by choice. 
 For example: 
 ``` .env
-TIMEZONE=“Etc/GMT+6”
+TIMEZONE=“”
```

Comment:
Why would empty string give us an output in GMT-0600?

Reply:
Yup, this was wrong. Fixed.